src/event.js

```diff
@@ -65,7 +65,9 @@
       handler.proxy = function(e){
         e = compatible(e)
         if (e.isImmediatePropagationStopped()) return
-        e.data = data
+        var dataPropDescriptor = Object.getOwnPropertyDescriptor(e, 'data')
+        if (!dataPropDescriptor || dataPropDescriptor.writable)
+            e.data = data
```

Comment:
It seems that when using strict mode `dataPropDescriptor` will be undefined when `e` is `InputEvent` (even though `data` property exists). Seems like a bug to me. To workaround this I would propose to wrap this line around a try/catch and add a comment about this.